### PR TITLE
Registration PR Kanishk Gupta

### PR DESCRIPTION
**Pull request by gkanishk**
Hey Tanay,
Excited for the challenge!! Please accept my PR

### CODE REVIEW

File: content/participants/ttjc@gkanishk/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+type: "participant"
+name: "Kanishk Gupta"
+portfolio: "https://gkanishk.github.io/"
+github-link: "https://github.com/gkanishk"
+linkedin-link: "https://linkedin.com/in/gkanishk"
+looking-for: "job"
+description: "Self learner good at web development,mobile development and working on react "
+---
+
+Hi! I am a Kanishk Gupta,Microsoft Student Partner at Chandigarh University let me walk you through my intro, projects and blogs.
```
Comment (tanaypratap) [inline content/participants/ttjc@gkanishk/index.md]: typo. `a` is not needed.

File: content/participants/ttjc@gkanishk/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### introduction
+
+Currently, a student pursuing B.E CSE Hons. in Cloud Computing at Chandigarh University, holding the title of Microsoft Student Partner.
```
Comment (tanaypratap) [inline content/participants/ttjc@gkanishk/index.md]: typo B.E. the next dot is missing

File: content/participants/ttjc@gkanishk/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Passionate about participating in competitions like Hackathons and organizing events.Keen interest in research on new technologies. I like contributing to the community and share my knowledge.
```
Comment (tanaypratap) [inline content/participants/ttjc@gkanishk/index.md]: space after full stop.

File: content/participants/ttjc@gkanishk/index.md (hunk 1 above)
Comment (gkanishk) [reply content/participants/ttjc@gkanishk/index.md]: Done

File: content/participants/ttjc@gkanishk/index.md (hunk 2 above)
Comment (gkanishk) [reply content/participants/ttjc@gkanishk/index.md]: Done

**Comment by gkanishk** [comment]
Done with the typos, Good to be merged ✌

**Pull request merged**